contrib/cmake/CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ set( CORTEX_SOURCE_DIR ${CMAKE_SOURCE_DIR}/../.. )
 
 file( READ  ${CORTEX_SOURCE_DIR}/SConstruct SCONSTRUCT )
 
+string( REGEX MATCH "ieCoreMilestoneVersion=([0-9]+)" ${SCONSTRUCT} )
+set( IECORE_MILESTONE_VERSION ${CMAKE_MATCH_1} )
```

Comment:
This CMake commit is totally untested by me... maybe @dboogert  or @boberfly can let me know if its working?